ops/charts/devnet/templates/wasmd-deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wasmd
+  labels:
+    app: wasmd
+    release: {{ .Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: wasmd
+  template:
+    metadata:
+      labels:
+        app: wasmd
+    spec:
+      containers:
+        - name: wasmd
+          image: "{{ .Values.wasmd.image.repository }}:{{ .Values.wasmd.image.tag }}"
+          imagePullPolicy: {{.Values.wasmd.image.pullPolicy}}
+          env:
+            - name: PASSWORD
+              value: cosm1234
+            - name: CHAIN_ID
+              value: testing
+            # If we need to change these for better control in testing we can
+            # - name: MONIKER
+            #   value: node001
+            # - name: STAKE
+            #   value: ustake
+            # - name: FEE
+            #   value: ucosm
```

Comment:
Not sure if `testing` is the needed chain id but it is what is set by default by the script. We can change more of this here as well as the other ones I left commented out.